indico/web/client/js/jquery/widgets/jinja/person_link_widget.js

```diff
@@ -381,6 +382,20 @@
       $field.principalfield('enter');
     });
 
+    $buttonAddMyself.on('click', function() {
+      indicoRequest('search.users', options.sessionUser, function(result, error) {
```

Comment:
Why not pass all the data you get in the result via `options.sessionUser` and thus completely avoid the legacy API request?

Reply:
Well, I'd say that the whole person_link_widget uses this legacy API so I just followed what was done in search. I understand that could be a better approach, but was not sure of how to build the right data for add so resorted to this call that seemed consistent with the rest. I do not feel I could do what you suggest, despite it seems reasonable, sorry!